lib/homeview.dart

```diff
@@ -1,27 +1,53 @@
-// Skapa hemvyn med to do listan
+// Hemvy med en to do list
 
 import 'package:flutter/material.dart';
+import 'package:template/secondview.dart';
 
 class MyHomePage extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      appBar: AppBar(
-        title: Text('To Do List'),
```

Comment:
En trevlig kodsnutt att lägga in här så man slipper den röda/gula bannern medans man testar sin kod är:
 debugShowCheckedModeBanner: false,